shared/output/cloud/secretmanager_v1beta1/lib/google/cloud/secret_manager/v1beta1/paths.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Auto-generated by gapic-generator-ruby. DO NOT EDIT!
+
+
+module Google
+  module Cloud
+    module SecretManager
+      module V1beta1
+        # Path helper methods for the google.cloud.secrets.v1beta1 API.
+        module Paths
+          ##
+          # Create a fully-qualified Secret resource string.
+          #
+          # The resource will be in the following format:
+          #
+          # `projects/{project}/secrets/{secret}`
+          #
+          # @param project [String]
+          # @param secret [String]
+          #
+          # @return [String]
+          def secret_path project:, secret:
+            raise ArgumentError, "project is required" if project.nil?
+            raise ArgumentError, "project cannot contain /" if %r{/}.match? project
```

Comment:
`include` is actually faster than `match` on recent Rubies ([source](http://www.mervine.net/ruby-methods-vs-regex)). If we're going to keep the regexp, I think it makes sense to make `%r{/}` a constant like `SLASH_RE = %r{/}.freeze` to avoid allocating a new regexp each time a path helper is called.